.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
       os: osx
       compiler: gcc
       before_script:
-        - mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE
+        - mkdir build && cd build && cmake .. -DCMAKE_BUILD_TYPE=Debug -DBUILD_TEST=TRUE
```

Comment:
Enable debug run ONLY on one of the runs without any of the sanitizers. You could use this one for example or the Linux one. We need our CI to be running with the release mode in general.

Reply:
Makes sense. I'll update it.